stellar-horizon/debian/stellar-horizon.postinst

```diff
@@ -36,11 +36,15 @@ case "$1" in
       if [ -f /etc/default/stellar-horizon ]; then
         # extract --db_url from /etc/default/stellar-horizon
         db_connection_string=$(grep -r '^DATABASE_URL' /etc/default/stellar-horizon | sed 's/DATABASE_URL=//g' | sed 's/"//g');
+        # extract INGEST flag from /etc/default/stellar-horizon. Default
+        ingest_flag=$(grep -r '^INGEST' /etc/default/stellar-horizon||echo "INGEST=true");
```

Comment:
Variable name should `INGEST`?

Reply:
I followed convention used for the `db_connection_string` here